controllers/configurationpolicy_controller.go

```diff
@@ -1058,12 +1078,22 @@ func (r *ConfigurationPolicyReconciler) handleSingleObj(
 	exists bool,
 	dclient dynamic.Interface,
 	objectT *policyv1.ObjectTemplate,
-) (objNameList []string, compliance bool, rsrcKind string, statusUpdateNeeded bool) {
+) (
+	objNameList []string,
+	compliance bool,
+	rsrcKind string,
+	statusUpdateNeeded bool,
+	creationInfo map[string]interface{},
```

Comment:
Optional:
```suggestion
	creationInfo policyv1.ObjectProperties,
```
Using the type might be a bit more performant, and would save some conversion later.